pkg/processor/build/runtime/python/test/reverser/reverser.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def handler(context, event):
+    """Return reversed body as string"""
+
+    try:
+
+        # decode the binary body into a string
+        body = event.body.decode('utf-8')
+
+        # return the reverse
+        return body[::-1]
+
+    except:
```

Comment:
Base `except` is usually not a good idea, it's better to catch `Exception` (since here you'll also catch `KeyboardInterrupt`.

Also log the error?

Reply:
I guess... but it's an integration test module